discli

```diff
@@ -243,6 +285,10 @@ class DiscourseHelper:
 
         raise RuntimeError(f"{topic} is neither a topic id nor a known filename")
 
+    def last_gulp(self):
+        info = self.dotfile_contents()
+        return info.get('_meta', {}).get("updated_at")
```

Comment:
Does this work if they are missing?

Reply:
Yes. `self.dotfile_contents()` returns an empty `dict` when the file isn't present.